### ISSUE TITLE
About footer

### ISSUE THREAD

**Issue by jhq223** [saicaca/fuwari#230]
![image](https://github.com/user-attachments/assets/27fcba59-f99c-456f-b0ba-890c40ac9f95)
Should the footer and pagination be centered at the bottom of the screen?

**Comment by L4Ph** [comment]
The footer can be centered if desired. However, when it comes to pagination, considering its role as pagination for blog content, it should not be placed in the center of the screen. If you want to center it, I think the blog content itself should be centered.
![image](https://github.com/user-attachments/assets/97ea6ad6-b5c1-4bcc-a208-8c49d319ca93)

**Comment by jhq223** [comment]
You are right, the pagination should indeed be consistent with the content layout. However, I think the footer should be centered at the bottom. @L4Ph

**Comment by saicaca** [comment]
I will see if centering looks better and will make the change if it does.
